src/layout/Content/content.util.js

```diff
@@ -21,10 +24,18 @@ export const generateGroupedItems = (content, selectedRowData) => {
         : (groupedItems[`${contentItem.name}-${contentItem.metadata.name}`] = [
             contentItem
           ])
-    } else {
+    } else if (contentItem.name) {
       groupedItems[contentItem.name]
         ? groupedItems[contentItem.name].push(contentItem)
         : (groupedItems[contentItem.name] = [contentItem])
+    } else if (contentItem.db_key) {
+      groupedItems[contentItem.db_key]
+        ? groupedItems[contentItem.db_key].push(contentItem)
+        : (groupedItems[contentItem.db_key] = [contentItem])
```

Comment:
I prefer:
```suggestion
      groupedItems[contentItem.db_key] ??= []
      groupedItems[contentItem.db_key].push(contentItem)
```
The first statement makes sure that `groupedItems[contentItem.db_key]` defaults to an array in case it is `undefined` or `null`. Then the second statement pushes to this array, whether it was just set as empty, or it was already a (non-empty) array before.

One more benefit to this is that I don't like the use of the ternary operator `?:` when the result of the expression is not used.

```js
const foo = cond ? a : b // here it is used to initialize a variable
const obj = {
  prop: cond ? a : b // here it used to initialize an object property
}
someMethod(cond ? a : b) // here it is used as a function call argument
const func = () => cond ? a : b // here it is used as the return value of a function call
const result = a + b / c + (cond ? e ** 10 : 3 + f) // here it is used as part of a more complex expression
const str = `bla bla ${cond ? ' (also bla)' : ''}` // here it is used as part of a more complex expression
```